### DISCUSSION TITLE
I signed in in vscode but dont work

### DISCUSSION THREAD

**Discussion by plaraje** [community/community#13791] [Copilot Conversations]
i've trying to conect to github copilot but suggestions dont appear

**Comment by davecheney** [comment]
Are you using VSCodium or OSS Code?

**Reply by plaraje** [reply]
VSCode

**Reply by plaraje** [reply]
> VSCode

Visual studio code

**Comment by altacountbabi** [comment]
same issue here

**Reply by davecheney** [reply]
Are you using VSCodium or OSS Code?

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by hkllin** [reply]
same issue here

**Reply by hkllin** [reply]
After I linked it, it didn't work

**Reply by davecheney** [reply]
> After I linked it, it didn't work

I don’t understand. Could you please explain what you did, what you expected, and what happened instead.

**Reply by hkllin** [reply]
With m1 vscode installed, the robot can't be used.

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by hkllin** [comment]
<img width="920" alt="image" src="https://user-images.githubusercontent.com/93277196/167388244-3026c1f4-47ad-4d42-9418-ed641009175b.png">
 
It can be used before I update vscode.

**Reply by hkllin** [reply]
<img width="1206" alt="image" src="https://user-images.githubusercontent.com/93277196/167389208-38e7efc2-b318-469f-a81d-9ca09046efbf.png">

**Reply by hkllin** [reply]
<img width="1279" alt="image" src="https://user-images.githubusercontent.com/93277196/167389710-e62d6499-2b6b-4292-83fb-f960de542580.png">

**Reply by davecheney** [reply]
You gotta show me the GitHub copilot tab, dawg

**Reply by plaraje** [reply]
Hello, thank you, although you have not helped me, thank you for trying, I have only reinstalled it and it did not ask me to log in but it has started to work for m
![#TH4NKS](https://user-images.githubusercontent.com/61209614/167460530-107ae103-3250-4129-abb6-a7f01a108483.PNG)
e

**Comment by hkllin** [comment]
<img width="1045" alt="image" src="https://user-images.githubusercontent.com/93277196/167522908-6ea8e8f2-209c-47e1-9ed2-97fd8e9dd4e0.png">
 this?

**Comment by hkllin** [comment]
<img width="1307" alt="image" src="https://user-images.githubusercontent.com/93277196/167523265-aafd0b7f-0f3a-4cb2-b6b4-2288e764cacb.png">
nothing...

**Reply by davecheney** [reply]
Thank you. That is the information I needed.

a. are you using VSCodium or OSSCode?
b. this might be an extension conflict; what other extensions are you using?

**Comment by hkllin** [comment]
I reinstalled it, but it still doesn't work.

**Reply by hkllin** [reply]
I downloaded Kanban Niang before, and then I uninstalled Kanban Niang, and then I couldn't use copilot.

**Comment by hkllin** [comment]
<img width="320" alt="image" src="https://user-images.githubusercontent.com/93277196/167523931-0457ec9b-3844-429f-a253-7ed0921047aa.png">
this 。。。 but i uninstalled it

**Comment by hkllin** [comment]
<img width="293" alt="image" src="https://user-images.githubusercontent.com/93277196/167524221-f23adb2a-10d0-42fa-98f8-365cace06287.png">
<img width="402" alt="image" src="https://user-images.githubusercontent.com/93277196/167524241-5bd68a6a-1271-42db-a29c-9dc0f956e345.png">

**Comment by hkllin** [comment]
I also tried uninstalling vs and downloading it again, but it didn't work.

**Reply by davecheney** [reply]
@hkllin i've checked the telemetry and your network is blocking you from using copilot. 
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Comment by hkllin** [comment]
@davecheney  My friend can use it, but I can't use it when I use mobile phone hotspots.

**Reply by davecheney** [reply]
What I am seeing is your computer is not able to connect to one or more of

- vscode-auth.github.com
- api.github.com
- copilot-proxy.githubusercontent.com

**Reply by hkllin** [reply]
i think 。。is my mac problems ...

**Comment by hkllin** [comment]
<img width="1142" alt="image" src="https://user-images.githubusercontent.com/93277196/167526236-ad2fd27b-9556-4d91-a9c0-d509c9e626b7.png">
